odoo/tools/mail.py

```diff
@@ -231,10 +233,14 @@ def html_sanitize(src, silent=True, sanitize_tags=True, sanitize_attributes=Fals
             'strip_classes': strip_classes,  # remove classes, even when keeping other attributes
         })
 
+    # Avoid hard limit of 254 nested levels on some situations
+    elm_count = src.count("<") - src.count("</")
```

Comment:
But this is not counting nested levels. Only the appearance of that symbols, nested or not. It can be a simplification, but change the comment text according this.

Reply:
The comment says "on some situations", just because of this. What would you change?

Reply:
As, said, this is not counting nested levels, but the appearance of that (nested or not). This HTML will be counted as 3 nested levels: `<a></a><a></a><a></a>` while there's no nesting. Then, the comment can be:
```python
# Check the number of tags as quick diagnose of possible nesting
```